### DISCUSSION TITLE
error import wagmi in plain html

### DISCUSSION THREAD

**Discussion by oleksiivinogradov** [wevm/wagmi#2149] [Q&A]
Inside index.html:
 <script type="module">
    import wagmi from 'https://cdn.jsdelivr.net/npm/wagmi/+esm'
  </script>

ils.js:1 Uncaught TypeError: Cannot read properties of null (reading 'AbiCoder')
    at Object.get [as AbiCoder] (utils.js:1:1)
    at utils.js:1:1
g

looks like wrong dependencies?

**Comment by tmm** [comment]
As of now, wagmi requires ethers as a peer dependency.

**Reply by oleksiivinogradov** [reply]
<script type="module">
    import ethers from 'https://cdn.jsdelivr.net/npm/ethers/+esm'
    import wagmi from 'https://cdn.jsdelivr.net/npm/wagmi/+esm'
  </script>

Still doesn't work

**Reply by oleksiivinogradov** [reply]
I also has   <script src="https://cdn.ethers.io/lib/ethers-5.4.umd.min.js" type="application/javascript" async></script>
